ui/src/hooks/useTerminalInstance.ts

```diff
@@ -170,8 +147,8 @@ export const useTerminalInstance = (socket?: WebSocket | '') => {
   };
 
   return {
+    fitAddon,
     setTerminalTheme,
-    terminalResizeEvent,
     createTerminalInstance
   };
 };
```

Comment:
In this code snippet, the `export` statements at the top of each function declaration are not valid JavaScript syntax. The comments and other sections look fine. However, since the last section appears to be an incomplete class definition, no code changes should be expected here.

For further review, there doesn't seem to be anything else that needs to be updated regarding style, structure, or functionality in these two files. This is assuming they're part of a larger project.

If you require detailed reviews on specific functions, variable names, naming conventions, etc., please share them with more context!